vizro-core/src/vizro/models/_components/ag_grid.py

```diff
@@ -35,7 +34,7 @@ class AgGrid(VizroBaseModel):
 
     type: Literal["ag_grid"] = "ag_grid"
     figure: CapturedCallable = Field(
-        ..., import_path=vt, mode="ag_grid", description="Function that returns a Dash AgGrid."
+        ..., import_path="vizro.tables", mode="ag_grid", description="Function that returns a Dash AgGrid."
```

Comment:
Would probably stick to their syntax

```suggestion
        ..., import_path="vizro.tables", mode="ag_grid", description="Function that returns a `Dash AG Grid`."
```